.env

```diff
@@ -4,6 +4,8 @@
 # Then just do this to import them:
 # import { API_URL } from 'react-native-dotenv'
 
+INIT_APP_TIMEOUT=10000
```

Comment:
10000 seems short, what is the average value we are expecting if there is no bug?

Reply:
less than a sec

Reply:
ok stick with 10000 then